### PR TITLE
Refactoring best practices/#2

### PR DESCRIPTION
**Pull request by Lirt**
Refactoring according to docker best practices.
This PR also replaces supervisor in favor of running postfix in foreground mode and adds posibility to use mounted docker volume with postfix configuration instead of docker environment variables.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,20 +1,38 @@
...
-ENV LANG en_US.utf8
+RUN apt-get update && apt-get install -y \
+  locales \
+  postfix \
+  syslog-ng \
+  syslog-ng-core 
 
-RUN apt-get update 
-RUN apt-get install -y postfix supervisor syslog-ng syslog-ng-core
+RUN rm -rf /var/lib/apt/lists/*
```
Comment (kirecek) [inline Dockerfile]: i'd move to this to upper layer (apt-get update & install ...)

File: Dockerfile [hunk 2]
```diff
@@ -1,20 +1,38 @@
-FROM debian:latest
+FROM debian:stretch
 MAINTAINER Ondrej Vasko
```
Comment (kirecek) [inline Dockerfile]: MAINTAINER is deprecated

https://docs.docker.com/engine/reference/builder/#maintainer-deprecated

File: Dockerfile (hunk 1 above)
Comment (Lirt) [reply Dockerfile]: Like above of apt-get?

File: Dockerfile [hunk 3]
```diff
@@ -3,13 +3,14 @@ FROM debian:stretch
 ENV DEBIAN_FRONTEND noninteractive
 ENV LANG en_US.utf8
 
+RUN rm -rf /var/lib/apt/lists/*
```
Comment (kirecek) [inline Dockerfile]: sorry, i've ment merge not just move:

```dockerfile
RUN apt-get update \
 && apt-get install -y \
    <packages> \
 && rm -rf /var/lib/apt/lists/*

```

**Comment by Lirt** [comment]
Ty @kirecek

**Pull request merged**